docs/control-structures.rst

```diff
@@ -250,7 +250,7 @@ use a different mechanism to come up with the address of the new contract:
 
 It will compute the address from the address of the creating contract,
 the given salt value, the (creation) bytecode of the created contract and the constructor
-arguments.
+arguments encoded using ``abi.encode``.
```

Comment:
While his is true, we don't specify how the other elements are used specifically to compute the address. Furthermore, as it is now, it is not clear if `abi.encode` only applies to the arguments or to the whole thing.

Maybe it's better to limit this change to only adding `abi.encode` in the code below?

Reply:
That's true... The only important change is the code example anyway, after re-reading the text, it is a more high level explanation, giving details on every encoding here would probably be too heavy to read and redundant with the code example.

Do you want another PR focusing only on the code change ?

Reply:
No need for a new PR, you can amend the commit ("git commit --amend") and then force-push ("git push --force").